doc/examples/Format_Features/Format_Features.ipynb

```diff
@@ -8,9 +8,9 @@
     "\n",
     "This notebook demonstrates minimal examples of the following format features:\n",
     "\n",
-    "* [Time-dependent right hand sides](.time_dependent_rhs.yml)\n",
-    "* [Step functions in the right hand side](step_function.yml)\n",
-    "* [Function definition](function_in_rhs.yml)"
+    "* Time-dependent right hand sides\n",
+    "* Step functions in the right hand side\n",
+    "* Function definition"
```

Comment:
```suggestion
    "* Function definitions"
```
Since it's referring to the format feature in general, not the use case in the notebook